src/modules/analysis/AnalysisAddComponent.jsx

```diff
@@ -55,29 +88,42 @@ function newLine() {
   </Form>;
 }
 
+const getVal = (form, field) => form[field] && (form[field].rawValue || ''); // TODO may be inline
```

Comment:
Remove it completely. As said in todo. It should be inlined or conveted to a function that takes a field and returns Either rawValuw or empty